src/wwwroot/locales/en-translation.js

```diff
@@ -181,8 +181,11 @@
   "resubscribe_label_reason_input": "Reason:",
```

Comment:
This is OK en English but in Spanish should be "Motivo" as the confluence documentation says: https://makingsense.atlassian.net/wiki/spaces/DR/pages/775389311/Rehabilitar+emails+bloqueados